queries/hjson/indents.scm

```diff
@@ -0,0 +1,3 @@
+; inherits: json
+
+(comment) @ignore
```

Comment:
Ignore?

Reply:
This is so comments aren't taken into consideration to calculate the current indentation level, in particular when the comment is a multiline comment.